core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -156,7 +156,9 @@ public Boolean apply(Void input) {
         try {
           return fs.getStatus(uri).isPersisted();
         } catch (Exception e) {
-          throw Throwables.propagate(e);
+          Throwables.propagateIfPossible(e);
```

Comment:
Thanks for the initiative of replacing the deprecated method!
Shall we use `throwIfUnchecked`? According to the [JavaDoc](https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/base/Throwables.html#propagateIfPossible-java.lang.Throwable-): `propagateIfPossible` is scheduled to be removed in July 2018.